common-graphql/src/main/resources/graphql/schemas/UserPreferencesDB.graphql

```diff
@@ -1570,19 +1571,18 @@ input lucuma_target_pk_columns_input {
 columns and relationships of "lucuma_target_preferences"
 """
 type lucuma_target_preferences {
-  agsCandidates: Boolean!
-  agsOverlay: Boolean!
-  fov: bigint
-  fovDec: bigint!
-  fovRA: bigint!
-  fullScreen: Boolean!
+  agsCandidates: Boolean
```

Comment:
There's a `fetchUserPreferencesSchema.sh` so that we can import the schema from Hasura and avoid us the burden of maintaining it manually.

Maybe there's a way to declare the fields as optional in Hasura?